debian/copyright

```diff
@@ -0,0 +1,15 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: *
+Copyright: 2021 Zbigniew Jędrzejewski-Szmek <zbyszek@in.waw.pl>
+           2021 Luca Boccassi <bluca@debian.org>
```

Comment:
Do we need that? In my experiency, keeping this up to date gets boring pretty quickly, and there's `git shortlog` which gives a much realerer picture anyway. And CC-0 means that there *is no copyright*. So maybe just say `Copyright: none` or something fake-to-a-human to appease linters?

Reply:
Yes, in case the implementation in dpkg/debhelper doesn't gain traction and I want to upload this to Debian/Ubuntu, I'd need this to pass FTP review. It doesn't need to be kept up to date, it's just to pass initial review.